frontend.py

```diff
@@ -79,7 +79,8 @@ def printAppUse():
 	printColor("2. Edit Note(Limited Functionality in v0.01)", getConfig(2),"")
 	printColor("3. Search Notes",getConfig(2),"")
 	printColor("4. List Notes",getConfig(2),"")
-	printColor("5. Delete Note",getConfig(2))
+	printColor("5. Delete Note",getConfig(2),"")
+	printColor("6. Convert Note to MKDown",getConfig(2))
```

Comment:
these two lines are conflicting with the frontend.py in main and they cause a merge conflict. **What is the difference between converting a note to MKDown and exporting to PDF**? Option 6 in the current version asks for exporting to PDF, here you have converting to markdown. Do we want 7 options, or do we just want you convert to MKDown option? I am not sure when exporting to PDF was added.

Reply:
I changed mine to option 7 so it would be easier to deal with merge conflicts.